src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/PublicationController.cs

```diff
@@ -73,6 +75,30 @@ public async Task<ActionResult<List<PublicationSummaryViewModel>>> ListPublicati
                 .HandleFailuresOrOk();
         }
 
+        [HttpGet("api/publication/{publicationId:guid}/external-methodology")]
+        public async Task<ActionResult<ExternalMethodologyViewModel>> GetExternalMethodology(Guid publicationId)
```

Comment:
This is a `Content.Services.ViewModels` view model, intended to be returned to the public. You've been able to import this after `Content.Services` was made a dependency of Admin for other fairly recent changes, but I think the intention still stands that Admin has its own view models.

By sharing view models with the public, it potentially exposes us to problems where a change to a common view model required for the Admin unknowingly exposes some private data to the public, or alters it in a way that breaks the public site.

It also opens us up to potential friction in the future when the view models do need to diverge.